### DISCUSSION TITLE
Add Brevo Integration

### DISCUSSION THREAD

**Discussion by DowleyDeveloped** [solspace/craft-freeform#1519] [Feature Requests]
Would be good to get Brevo integrated with Freeform, as it's quite a popular email marketing platform.

**Comment by DowleyDeveloped** [comment]
@kjmartens Morning (From Uk!)

Just wondering if anyone had any thoughts on this? Requests are becoming more frequent for Brevo intregation from clients as it's starting to grow so just wondering if this was in your future plans or not?

**Comment by kjmartens** [comment]
Hi @DowleyDeveloped,

It's currently not on our immediate plans to support, but we can have a look at the API and make a decision sometime in the not-too-distant future.

In the meantime, we also do offer custom development services where we can expedite the addition of an integration like this and ensure it works for your needs. If you're interested in that, please create a support ticket or email us at support@solspace.com for more info. 🙂

**Comment by white-willem** [comment]
@kjmartens We have around 11 clients who would greatly benefit from a Brevo integration as well. Could you please take this into consideration again? I look forward to hearing from you.

**Comment by kjmartens** [comment]
@DowleyDeveloped and @white-willem 

Can you describe which features you need specifically from it? Would this be a standard integration like Mailchimp or Campaign Monitor, etc? 🙂

**Reply by white-willem** [reply]
@kjmartens Thanks for the quick response. Yes exactly like the standard integration with Mailchimp is what we are looking for. So subscribing to lists and mapping fields to Brevo fields.

**Reply by kjmartens** [reply]
Thanks @white-willem,

We've queued this up now and should probably have it available sometime next week. 🙂

**Reply by white-willem** [reply]
@kjmartens Awesome! Thanks for the quick follow up.

**Comment by seandelaney** [comment]
https://github.com/solspace/craft-freeform/pull/2553

**Comment by kjmartens** [comment]
This is now available in Freeform 5.15.17+ 🙂

**Reply by white-willem** [reply]
Thanks for the quick realisation of this feature @kjmartens. Much appriciated!
